src/App.css

```diff
@@ -1,38 +1,572 @@
-.App {
+/* page body */
+
+body {
   text-align: center;
+  background: rgb(255, 254, 143);
+  background: linear-gradient(
```

Comment:
Multiple definitions for the same rule just result in the last value being used.